index.js

```diff
@@ -63,6 +66,12 @@ module.exports = (bookshelf, settings) => {
     initialize: function () {
       modelPrototype.initialize.call(this)
 
+      if (settings.sentinel) {
```

Comment:
Users should also be able to activate and deactivate the sentinel feature in each model